apps/cli/src/commands/profile/create.profile.ts

```diff
@@ -82,24 +82,75 @@ export default class CreateProfile extends BaseCommand {
   }> {
     let { name, apiKey, baseUrl, setDefault, enableMetrics } = options
 
-    if (!name) {
+    if (
+      !name &&
+      !apiKey &&
+      !baseUrl &&
+      setDefault === undefined &&
+      enableMetrics === undefined
+    ) {
       name = await text({
         message: 'Enter the name of the profile',
         placeholder: 'work'
       })
-    }
 
-    if (!apiKey) {
       apiKey = await text({
         message: 'Enter the API key for the profile',
         placeholder: 'ks_************'
       })
-    }
 
-    if (!enableMetrics === undefined) {
+      baseUrl =
+        (await text({
+          message: `Enter the base URL for the keyshade server (default: ${API_BASE_URL})`,
+          placeholder: API_BASE_URL
+        })) || API_BASE_URL
+
+      setDefault = await confirm({
+        message: 'Set the profile as the default profile?',
+        initialValue: false
+      })
+
       enableMetrics = await confirm({
-        message: 'Should keyshade collect anonymous metrics for development?'
+        message: 'Should keyshade collect anonymous metrics for development?',
+        initialValue: false
       })
+    } else {
+      if (!name) {
+        name = await text({
+          message: 'Enter the name of the profile',
+          placeholder: 'work'
+        })
+      }
+
+      if (!apiKey) {
+        apiKey = await text({
+          message: 'Enter the API key for the profile',
+          placeholder: 'ks_************'
+        })
+      }
+
+      if (baseUrl === undefined) {
```

Comment:
I don't think we need these blocks in here. The goal of this block is to not prompt the users to send in values that are optional.